init/common.js

```diff
@@ -202,6 +206,19 @@ const createProject = async (interestId, userId, project, addTeam=true) => {
   });
 };
 
+const createOpportunity = async (interestId, opportunity) => {
+    let existing = await strapi.entityService.findMany("api::opportunity.opportunity", {filters: {title: opportunity.title}});
+    existing = existing[0];
+    if (existing) {
+      return;
```

Comment:
I don't know very well how it works. This is a question, why is it necessary to scape if `existing` ? Should we trown an error in this case ?

Reply:
We are trying to avoid creating opportunity if one with the same title already exists. We are just generating test data here so that all tests can use it if required.